### DISCUSSION TITLE
Search property?

### DISCUSSION THREAD

**Discussion by QuentinFayt** [josdejong/svelte-jsoneditor#481] [Q&A]
Hi!
We are using your project with VueJs, and I noticed that we could use the MainMenuBar to search for values in the JSON provided to the component.
Is there a way to bypass that feature and provide a string to search directly to the component?
We are using your project as an easy way to display JSON to users, not edit it. In that context, the MainMenuBar isn't really needed, and we would like to render JSON with the user's search results (as it is rendered when the user performs a query).

**Comment by josdejong** [comment]
So I guess you would like to have an API to control the search box? 

You can make the editor `readOnly`, that will automatically hide most buttons in the main menu bar. You can also hide other buttons and only keep the search button for example, and custom style the menu bar if you want.
